.ci/install.sh

```diff
@@ -70,7 +70,7 @@ set_filename() {
 
 download_fnm() {
   if [ "$USE_HOMEBREW" == "true" ]; then
-    brew install Schniz/tap/fnm > /dev/null
+    brew install fnm > /dev/null
```

Comment:
Let's remove the `> /dev/null` because there are no caveats now 😄 
Then we'll see what Homebrew is telling us